webapp-django/crashstats/crashstats/static/crashstats/js/socorro/tablesorter_utils.js

```diff
@@ -0,0 +1,35 @@
+(function ($) {
+  $.tablesorter.addParser({
+    id: 'hexdigit',
+    is: function () {
+      return false;
+    },
+    format: function (str) {
+      var newstr;
+
+      if (str == null || str == '') {
```

Comment:
nit: use `===` especially with `null` check

Reply:
Oo--thank you! I'm surprised that this wasn't picked up in linting. Maybe I've got linting set up wrong.